Assets/Scenes/DonTestScenes/TestLevel03.unity

```diff
@@ -4915,7 +4915,7 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 66c96970ebf484ed1ac4e41deb8611bf, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  playerPrefab: {fileID: 2814348810090761049, guid: b81ceb76fe5494d449726a59c7cd6d4d, type: 3}
+  playerPrefab: {fileID: 2806582366043827928, guid: b81ceb76fe5494d449726a59c7cd6d4d, type: 3}
```

Comment:
For some reason the PlayerPrefab was `missing` in all of the test scenes. We should be careful when altering / moving prefabs as this can break existing references.